File: app/lib/WebpackBuildFile.scala
package lib

import java.io.File

object WebpackBuildFile {
  private val prefix = new java.io.File(".").getCanonicalPath
  private val d = new File(prefix+"/public/bundle")

  val jsBundle: String = if(d.exists && d.isDirectory) {
    d.listFiles.filter(_.isFile).toList.find(f => f.getName.contains("js.bundle.")).get.getName.replace(".gz", "")
  } else ""

//  val cssBundle: String = if(d.exists && d.isDirectory) {
//    d.listFiles.filter(_.isFile).toList.find(f => f.getName.contains("style.bundle.")).get.getName.replace(".gz", "")
//  } else ""

}

File: app/lib/WebpackBuildFile.java
package lib;

import java.io.File;
import java.io.IOException;


public class WebpackBuildFile {
	static String jsBundle = "";

	static {
		try {
			final String prefix = new File(".").getCanonicalPath();
			final File directory = new File(prefix + "/public/bundle");
			final File[] listOfFiles = directory.listFiles();

			if(listOfFiles != null) {
				for(final File file : listOfFiles) {
					final String fileName = file.getName();
					if(file.isFile() && fileName.contains("js.bundle.") && fileName.substring(fileName.lastIndexOf(".") + 1).equals("js")) {
						jsBundle = fileName;
					}
				}
			}

		} catch(final IOException e) {
			e.printStackTrace();
		}
	}

	public static String getJsBundle() {
		return jsBundle;
	}
}
